cmd/sifnoded/cmd/migrate_test.go

```diff
@@ -27,10 +27,6 @@ func TestMigrateGenesisDataCmd(t *testing.T) {
 	cmd.SetArgs([]string{"init", "test", "--home=" + homeDir})
 	err = svrcmd.Execute(cmd, homeDir)
 	require.NoError(t, err)
-	cmd, _ = NewRootCmd()
-	cmd.SetArgs([]string{"unsafe-reset-all"})
-	err = svrcmd.Execute(cmd, homeDir)
-	require.NoError(t, err)
```

Comment:
@Jedi2002 is there a reason we are remove those tests?

Reply:
The tests are still functional , just removed `unsafe-reset-all` as that command is not needed , and it has been deprecated